tests/e2e/tidbcluster/serial.go

```diff
@@ -783,6 +809,13 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 				if err != nil {
 					return false, err
 				}
+				// check readyToScaleTimestamp
+				if time.Now().Sub(time.Unix(readyToScaleTimestamp, 0)).Seconds() < 40 {
+					return true, fmt.Errorf("tikv seconds scale doesn't meet the ReadyToScale threshold")
```

Comment:
return false, fmt.Errorf("tikv seconds scale doesn't meet the ReadyToScale threshold")